packages/components/src/overlays/Popover/tests/chromatic/Popover.stories.tsx

```diff
@@ -120,3 +120,14 @@ export const Styling = {
         </PopoverTrigger>
     )
 } satisfies Story;
+
+export const NoHeading = {
```

Comment:
from what I remember, you are still working on removing the extra margin, right?

Reply:
Yes! Pushed a fix